modules/installation-mirror-repository.adoc

```diff
@@ -66,6 +102,7 @@ Success
 Update image:  <local_registry_host_name>:<local_registry_host_port>/<local_registry>/<local_repository>:<release_version>
 Mirror prefix: <local_registry_host_name>:<local_registry_host_port>/<local_registry>/<local_repository>
 
+ifdef::install[]
 To use the new mirrored repository to install, add the following section to the install-config.yaml:
 
 imageContentSources:
```

Comment:
nit, do you want a leading `----` before this literal block?

Reply:
Git is hiding the `----` for this codeblock.